### ISSUE TITLE
Forcing Horizontal doesn't work

### ISSUE THREAD

**Issue by jtagcat** [benji300/joplin-note-tabs#52]
![image](https://user-images.githubusercontent.com/38327267/129368637-cd0cbb8b-1012-4cab-998c-cd956d10ae28.png)

What info can I provide?

**Comment by benji300** [comment]
If you mean the vertical layout doesn't work anymore, I can confirm this issue.
Switching to the vertical layout, either automatically or with `Force tabs layout` set to `Vertical` has no effect.
I will analyze the root cause and will fix in in the next release.

Thanks for reporting 👍

**Comment by benji300** [comment]
Sorry @jtagcat, but I'm getting a little ahead of myself here. I'm afraid I can't understand what you mean exactly. 
Switching between horizontal and vertical layout works fine for me. No matter if automatically or via the `Force tabs layout` option. I have tested with Joplin v2.2.7 and the plugin version v1.4.0.

Please also note that if the option `Force tabs layout` is toggled, the changes will only become active after a restart of the app.

So could you please be a little more specific about what you are experiencing and what kind of behavior you would actually expect?

**Comment by jtagcat** [comment]
I have restarted the app, many times.

I can have a vertical layout, where the tabs are in a list, above-below themselves.

Setting it to automatic, it is always vertical, no matter I do.

Setting it to force horizontal (where tabs are left-right of each other) used to not work at all (keeping vertical).

***

Right now, it worked sub-optimally as seen on the image above: I would assume the tabs would be above the title of the note. ..and they can be. I checked the README again for reference, I didn't know there was a 'change application layout' button hidden in the drop-downs.

Pardon for troubling, pebcak.
